tests/periph_timer_short_relative_set/Makefile

```diff
@@ -0,0 +1,13 @@
+include ../Makefile.tests_common
+
+FEATURES_REQUIRED = periph_timer
+
+USEMODULE += core_thread_flags
+
+# optionally configure timer under test. Defaults to xtimer config.
+#CFLAGS += -DTEST_TIMER_DEV=foo -DTEST_TIMER_FREQ=bar
```

Comment:
a description on what kind of values /units are expected for the define values would be nice. E.g. "TEST_TIMER_DEV takes a 0-based index to the periph_timer instance defined in..." and "frequency as an integer number in Hz" or something like that.